### DISCUSSION TITLE
(Suggestion) New way to categorize and classify everything

### DISCUSSION THREAD

**Discussion by John-Enigma** [mamedev/discussions#28] [Ideas]
I find the way to deal (and navigate) through MAME a little bit too much in general. Everything is massively condensed in drop down menus, that there doesn't exist an option to truly classify everything. 

I would love to suggest a new way to organize everything, and that is by their own categories: Arcade Boards (Fighting Games, Shmups, Beat-Em-Ups, Racing Games), Machines (Not necessarily related to arcade games or consoles), and Consoles (especially for obscure consoles, like the Super A'can, or the Mattel Hyperscan; obscure handhelds like Hartung Game Master, or the Gamate; And Plug-N-Plays, like the majority of the Jakks-Pacific ones, or the similar looking ones). 

I took notice of problems like these, because in the Input Assignment section for Player 1, you'd see that everything is not classified in their own categories. It kinda looks like there's an input for pretty much everything arcade-related imaginable, that you can't figure out which is the one to use, and which isn't. It doesn't help that there aren't diagrams/pictures to guess what to assign, which you'd normally see in other emulators.

**Comment by ghost** [comment]
MAME does not really ship with subjective data, trying to categorize games is subjective, and best left to others.
